### PR TITLE
[IMP] developer/cli: add documentation for the new populate

### PR DESCRIPTION
**Pull request by Aurelienvd**

### CODE REVIEW

File: content/developer/reference/cli.rst [hunk 1]
```diff
@@ -739,6 +739,39 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
 
 This will create module *my_module* in directory */addons/*.
 
+.. _reference/cmdline/populate:
+
+Database Population
```
Comment (AntoineVDV) [inline content/developer/reference/cli.rst:744]:
```suggestion
Database population
```

File: content/developer/reference/cli.rst [hunk 2]
```diff
@@ -739,6 +739,39 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
...
+===================
+
+.. program:: odoo-bin populate
+
+Odoo Populate allows to duplicate existing data in a given database. This can be used
+for testing and benchmarking, when large tables are needed. The duplication process
```
Comment (AntoineVDV) [inline content/developer/reference/cli.rst:750]:
```suggestion
for testing and benchmarking when large tables are needed. The duplication process
```

File: content/developer/reference/cli.rst [hunk 3]
```diff
@@ -739,6 +739,39 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
...
+introduces variation for some fields, to respect `UNIQUE` constraints among other things.
```
Comment (AntoineVDV) [inline content/developer/reference/cli.rst:751]:
```suggestion
introduces variation for some fields to respect `UNIQUE` constraints, among other things.
```

File: content/developer/reference/cli.rst [hunk 4]
```diff
@@ -739,6 +739,39 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
...
+    list of models to populate. Models appearing twice will only be populated once.
+
+.. option:: --factors
+
+    list of populate factors. In case a factor is missing for a model, the last factor in
+    the list will be used.
+
+.. option:: --sep
+
+    separator used to generate a record's name
```
Comment (AntoineVDV) [inline content/developer/reference/cli.rst:773]:
```suggestion
    separator used to generate record names.
```

**Review by AntoineVDV — APPROVED** [review]
@robodoo delegate+

**Comment by Feyensv** [comment]
robodoo squash

**Comment by Aurelienvd** [comment]
@robodoo r+

**Pull request closed without merging**